website/react-magma-docs/src/pages/api/native-select.mdx

```diff
@@ -0,0 +1,187 @@
+---
+pageTitle: Native Select API
+title: Native Select
+props:
+  - NativeSelectProps
+---
+
+<DocsHeading type="api">NativeSelect</DocsHeading>
+
+## Basic Usage
+
+`Native Select` uses the magma <Link to="/api/select">`Select`</Link> styling with a browser default option menu for compatibility purposes.
+
+```typescript
+import React from 'react';
+import { NativeSelect } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <NativeSelect labelText="Select this">
+      <option>Red</option>
+      <option>Green</option>
+      <option>Blue</option>
+    </NativeSelect>
+  );
+}
+```
+
+## Disabled
+
+```typescript
+import React from 'react';
+import { NativeSelect } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <NativeSelect labelText="Select this" disabled>
+      <option>Red</option>
+      <option>Green</option>
+      <option>Blue</option>
+    </NativeSelect>
+  );
+}
+```
+
+## Error
+
+```typescript
+import React from 'react';
+import { NativeSelect } from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <NativeSelect
+      labelText="Select this"
+      hasError
```

Comment:
The consumer shouldn't need to specify `hasError` because they're including an `errorMessage` prop.